src/SUMMARY.md

```diff
@@ -17,6 +17,7 @@
     * [Event Pipeline Detail](concepts/pipeline/event_pipeline.md)
   * [Analysis Interfaces](tools/interfaces.md)
   * [Custom analysis with Spark](tools/spark.md)
+      * [Telemetry schema reference](tools/handy_schema.md)
```

Comment:
yeah, I ended up putting this under the "Custom Analysis with Spark" because there's a section in that page that specifically talks about the moztelemetry library, but leaves the reader wanting. Maybe we need a 3rd opinion ;)